frontend/packages/ceph-storage-plugin/src/components/dashboards/common/capacity-breakdown/consts.ts

```diff
@@ -15,3 +15,7 @@ export const COLORMAP = [
 export const OTHER = 'Other';
 export const CLUSTERWIDE = 'Cluster-wide';
 export const BUCKETCLASSKIND = 'BucketClass';
+export const DANGER = 'Danger';
+export const WARNING = 'Warning';
```

Comment:
```suggestion
export const DANGER = 'Danger';
export const WARNING = 'Warning';
```
These need to be translated as well.

Reply:
these are constants... I am not displaying them anywhere, I am using it for checking ternary conditions...
By "translated" do you mean: `export const DANGER = 'danger'`??